### ISSUE TITLE
ModelFieldImage shows a black bg when PNG file is chosen

### ISSUE THREAD

**Issue by yayosita** [ISBX/isbx-loopback-cms#155]
ModelFieldImage shows a black background when selecting a PNG file. In ImageService, the PNG file is passed as a jpeg type in data URL.

<img width="453" alt="screen shot 2017-05-27 at 7 00 52 pm" src="https://cloud.githubusercontent.com/assets/24953272/26520956/b5242750-430f-11e7-94e8-884593c06e8b.png">

**Comment by tma-isbx** [comment]
Specifically, a PNG with transparency. I think the canvas need a background color filled in with a default color.

**Comment by yayosita** [comment]
Yup, but in ImageService line 54, the data is always returned as a flat jpeg file so there's no way to fill the background color.
